beater/beater.go

```diff
@@ -37,11 +36,13 @@ func (bt *beater) Run(b *beat.Beat) error {
 	}
 	defer pub.Stop()
 
+	go notifyListening(bt.config, pub.Send)
```

Comment:
Could we trigger this after `err = run(bt.server, bt.config.SSL)` to ensure the server is up instead of polling for 10 seconds. 
The problem I see with the current handling is that the server has to be started within n seconds otherwise we never send a document to ES.

Reply:
As `run` is blocking, we can't start it afterwards. I think it's pretty save that the server is starting in 10s or otherwise returns an error (I hope ).